### PR TITLE
End-to-end city generator

### PR DESCRIPTION
**Pull request by NanyiJiang**
clutter

### CODE REVIEW

**Comment by NanyiJiang** [comment]
city

File: Assets/OBJImport/OBJLoader.cs [hunk 1]
```diff
@@ -0,0 +1,461 @@
+﻿/*
```
Comment (NanyiJiang) [inline Assets/OBJImport/OBJLoader.cs]: external library, dont need to review

File: Assets/OBJImport/TextureLoader.cs [hunk 2]
```diff
@@ -0,0 +1,166 @@
+﻿/*
```
Comment (NanyiJiang) [inline Assets/OBJImport/TextureLoader.cs]: external library, no need to review

File: Blender/GenerateBuilding.py [hunk 3]
```diff
@@ -342,7 +343,7 @@ def generate_building_base(bm,
     # Recenter the object to its center of mass
     bpy.ops.object.origin_set(type='ORIGIN_CENTER_OF_MASS')
     ob = bpy.context.object
-    ob.location = (0, 0, 0)
+    # ob.location = (0, 0, 0)
```
Comment (NanyiJiang) [inline Blender/GenerateBuilding.py]: @questmas i think the bulding is still off ground level

Comment (questmas) [reply Blender/GenerateBuilding.py]: I need to look into this. It might be that the buildings are growing downwards as well

**Pull request merged**